mezzanine/project_template/fabfile.py

```diff
@@ -333,6 +388,29 @@ def manage(command):
     return run("%s %s" % (env.manage, command))
 
 
+###########################
+# Security best practices #
+###########################
+
+@task
+@log_call
+@hosts(["root@%s" % host for host in env.hosts])
```

Comment:
For the `secure` task: yes, we're assuming root access. The idea is you only need this task when you receive a brand new VPS (which means you get the root password in an email). If you already have a VPS with a regular user with sudo privileges, you can skip this task completely and use the others with said user. If you are in a shared hosting environment where sudo is not available, then this fabfile is not really for you :P